spec/parity/usage_spec.rb

```diff
@@ -1,4 +1,4 @@
-require File.join(File.dirname(__FILE__), '..', 'lib', 'parity')
+require File.join(File.dirname(__FILE__), '..', '..', 'lib', 'parity')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.